packages/gestalt/src/Video/VideoControls.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+
+import * as React from 'react';
+import Box from '../Box/Box';
+import Icon from '../Icon/Icon';
+import Text from '../Text/Text';
+import Touchable from '../Touchable/Touchable';
+import VideoPlayhead from './VideoPlayhead';
+import PropTypes from 'prop-types';
+
+type Props = {|
+  currentTime: number,
+  duration: number,
+  fullscreen: boolean,
+  muted: boolean,
+  paused: boolean,
+  seek: (time: number) => void,
+  toggleFullscreen: () => void,
+  toggleMute: () => void,
+  togglePlay: () => void,
+|};
+
+const timeToString = (time?: number) => {
+  const rounded = Math.floor(time || 0);
+  const minutes = Math.floor(rounded / 60);
+  const seconds = rounded - minutes * 60;
+  const minutesStr = minutes < 10 ? `0${minutes}` : minutes;
+  const secondsStr = seconds < 10 ? `0${seconds}` : seconds;
+  return `${minutesStr}:${secondsStr}`;
+};
+
+export default function VideoControls(props: Props) {
+  const {
+    currentTime,
+    duration,
+    fullscreen,
+    paused,
+    muted,
+    seek,
+    toggleFullscreen,
+    toggleMute,
+    togglePlay,
+  } = props;
+  return (
+    <Box
+      position="absolute"
+      bottom
+      left
+      right
+      width="100%"
+      padding={2}
+      display="flex"
+      alignItems="center"
+    >
+      <Box padding={2}>
+        <Touchable onTouch={togglePlay} fullWidth={false}>
+          <Icon
+            accessibilityLabel=""
```

Comment:
@pfarejowicz for all of these, we should be providing good `accessibilityLabel`s passed in as props. Right now it just reads `button`, `button`, `button`